src/main/java/com/aws/greengrass/mqtt/bridge/BridgeConfig.java

```diff
@@ -323,6 +323,7 @@ public boolean reinstallRequired(BridgeConfig newConfig) {
                 || !Objects.equals(getMqttVersion(), newConfig.getMqttVersion())
                 || !Objects.equals(getSessionExpiryInterval(), newConfig.getSessionExpiryInterval())
                 || !Objects.equals(getMaximumPacketSize(), newConfig.getMaximumPacketSize())
-                || !Objects.equals(getReceiveMaximum(), newConfig.getReceiveMaximum());
+                || !Objects.equals(getReceiveMaximum(), newConfig.getReceiveMaximum())
+                || !Objects.equals(getMqtt5RouteOptions(), newConfig.getMqtt5RouteOptions());
```

Comment:
not for this PR, but we should think how to optimize this. technically this is only applicable to local mqtt source, but customers can still define route options for iot core source, even though it wouldn't have an effect..... that case would case reinstall, which we wouldn't want.